src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,61 @@
+import '../../App.css';
+import Button from '../../common/Button/Button';
+import { useNavigate } from 'react-router-dom';
+import getCourseDuration from '../../helpers/getCourseDuration';
+import Header from '../Header/Header';
+import formatCreationDate from '../../helpers/formatCreationDate';
+import PropTypes from 'prop-types';
+
+function CourseInfo(props) {
+	let navigate = useNavigate();
```

Comment:
ideally, when you're not reassigning the value, you should use const - that way it's clear to the reader what does, and what does not get reassigned